flepimop/gempyor_pkg/src/gempyor/testing.py

```diff
@@ -275,6 +281,62 @@ def sample_fits_distribution(
         return bool(np.greater(sample, 0.0))
 
 
+def run_test_in_separate_process(
+    script: str | Path, dest: Path | None = None, args: Iterable[str] = ()
+) -> None:
```

Comment:
I don't think we need to return the whole subprocess, but seems reasonable to provide back the return code, *especially* if we some day want to actually use return codes and then want to test obtaining specific errors.